apps/account-api/test/setup/package.json

```diff
@@ -5,6 +5,8 @@
   "main": "index.ts",
   "scripts": {
     "main": "tsx index.ts",
+    "test-revoke": "tsx test-revoke.ts",
+    "test-revoke-e2e": "tsx test-revoke-e2e.ts",
```

Comment:
Do we need these?
Maybe move `test-revoke.ts` to the Lore repo, although the one for `retireMsa` is already there; same flow, different extrinsic.

Reply:
But isn't that e2e logic already in `delegations.controller.e2e-spec.ts`?